src/mumble/AudioInput.cpp

```diff
@@ -1017,7 +1017,7 @@ void AudioInput::encodeAudioFrame(AudioChunk chunk) {
 		rnnoise_process_frame(denoiseState, denoiseFrames, denoiseFrames);
 
 		for (int i = 0; i < 480; i++) {
-			psSource[i] = denoiseFrames[i];
+			psSource[i] = qMin(qMax(denoiseFrames[i], -32768.0f), 32767.0f);
```

Comment:
We should prefer `std::min` and `std::max` over the Qt-specific functions

Also what are these numbers? Are they the minimum and maximum value representable by   the used data type? If so, I think it would be more clear, if we used `std::numeric_limits` to obtain these numbers.

Reply:
Hello, I have pushed the changes you requested. The reason I preferred using Qt-specific functions and number literals in my change was because that's what the rest of the file was doing. On a side-note, if we could use C++17 extensions I could have used `std::clamp` which is more elegant than `std::min(std::max())`.

Reply:
Yes indeed. However, in the long run we aim at decoupling ourselves a bit more from Qt (except for the actual UI stuff) so this is why we nowadays prefer using std-functions :)